client/components/account-status/account-tools/index.tsx

```diff
@@ -27,7 +29,13 @@ const handleReset = () => {
 };
 
 export const AccountTools: React.FC< Props > = ( props: Props ) => {
-	const accountLink = props.accountLink;
+	let accountLink = props.accountLink;
+	const isPoInProgress = props.poEnabled && ! props.poComplete;
+	if ( isPoInProgress ) {
```

Comment:
@oaratovskyi As I've shared here (p1712757976717789/1712736009.166459-slack-C03KTTK2YMA), I would keep the Finish setup button logic only concerned with initial KYC finished (details_submitted) and not concerned with full KYC for PO. If the initial KYC is completed, we shouldn't be showing the Finish setup button.

Reply:
Ok, sounds good